doc_source/source/admin/install_docker.rst

```diff
@@ -92,6 +92,11 @@ This is due to dependency caching, since the first build is what builds and cach
 Running the Containers (Development)
 ====================================
 
+**Note**: Before running the containers, ensure that the necessary directories are created on the host machine with the
+correct permissions. You can manually create the directories by running the following script:
+
+    ./scripts/globus/setup_collection_directory.sh all
+
```

Comment:
Maybe use code block syntax for the .rst format. Not sure why it wasn't used elsewhere.

.. code-block:: language

    <indented code goes here>

Reply:
Updated. Here's the new artifact. For now I'm going to consider this tech-debt and write up a ticket for it to avoid scope creep.

Read that the approach used before `::`

> is inadvisable unless also paired with a parser directive resembling .. highlight:: console at the head of your ReST document. Doing so informs the ReST parser (e.g., Sphinx) of the default language to be used for all code blocks lacking an explicit language. In this case, that should be console – the "language" associated with Bash shell one-liners intended to be run interactively. That said, explicit is better than implicit. [[ref](https://stackoverflow.com/questions/16397794/how-to-show-terminal-shell-code-snippets-in-sphinx#comment89061123_16416025)]



https://github.com/user-attachments/assets/3dca37d9-9059-4adb-8762-4b60fa134ae9